### ISSUE TITLE
v1.1.0 debian incompatibility (lib version range)

### ISSUE THREAD

**Issue by theUm** [confluentinc/confluent-kafka-go#360]
Description
===========
On debian there is single way to get latest version of librdkafka - is to install librdkafka lib that has range of versions, including latest, `0.11.6-1.1` for example:
https://packages.debian.org/source/buster/librdkafka

I've dig some more info and found out that `c` code of confluentkafka-go seeks version constnant in `librdkafka` `c` header file, and (I believe) due to "range" nature of librdkafka lib on debian this constant is commented out. File  `/usr/include/librdkafka/rdkafka.h`
```
#define RD_KAFKA_VERSION  0x000b06ff
```
Code, responsible for version checking is in go file `00version.go`.

#### Solution
For me, the solution was to switch from `debian` to `alpine` (containerization ftw!), but I believe there may be people that may face same issue sometime and this will be not the solution they accept.

#### Debug log:
```
/go/pkg/mod/github.com/confluentinc/confluent-kafka-go@v1.1.0/kafka/00version.go:44:2: error: #error "confluent-kafka-go requires librdkafka v1.1.0 or later. Install the latest version of librdkafka from the Confluent repositories, see http://docs.confluent.io/current/installation.html"
 #error "confluent-kafka-go requires librdkafka v1.1.0 or later. Install the latest version of librdkafka from the Confluent repositories, see http://docs.confluent.io/current/installation.html"
  ^~~~~

```

How to reproduce
================

check out my repo I created just to reproduce it. Requires docker
https://github.com/theUm/librd-debian-test

Checklist
=========
Please provide the following information:

 - [x] confluent-kafka-go and librdkafka version (`latest stable - v1.1.0`):
 - [x] Apache Kafka broker version: irrelevant
 - [x] Client configuration: `ConfigMap{...}` irrelevant
 - [x] Operating system: debian. tested on `stretch `and `buster`
 - [x] Provide client logs (with `"debug": ".."` as necessary)
 - [x] Provide broker log excerpts irrelevant
 - [ ] Critical issue

**Comment by edenhill** [comment]
Debian/Ubuntu typically does not provide the latest version of librdkafka, you should instead use the Confluent debian repositories as instructed in the error message:
` #error "confluent-kafka-go requires librdkafka v1.1.0 or later. Install the latest version of librdkafka from the Confluent repositories, see http://docs.confluent.io/current/installation.html"`

**Comment by edenhill** [comment]
http://docs.confluent.io/current/installation.html

**Comment by theUm** [comment]
I wanted just to use golang kafka consumer with `confluent-kafka-go`. I did not found suitable images for that purpose among the images listed here https://hub.docker.com/u/confluentinc/
I presume those images are for kafka itselt and I did not found `librdkafka` in them. 

Installing 670 mbs of all the confluent stack with `apt-get install confluent-platform-2.12` from manual also did not helped as there is no `librdkafka` lib dependency on those deb packages.

What I'm gonna say is that info from readme here https://github.com/confluentinc/confluent-kafka-go/blame/master/README.md#L133 is misleading, as one does not simply install `librdkafka-dev` on `debian` with `go` and use this good (really, thank you this package is amazing) package.

All works ok on alpine though
